Docs: tweaks to content code

patrickhlauke opened the pull request:
- add missing angle brackets to `<code>`
- make code block example actually multi-line, still add `<code>` as it's semantically code we're showing

patrickhlauke committed:
Missing angle brackets

patrickhlauke committed:
Fix code block example

use `<code>` (as it's still semantically code that we're marking up) and
make the example actually multiple lines...

cvrebert commented:
This would remove the only non-Reboot example of using `<pre>` without `<code>`. Seems like we should include some example of that case.

patrickhlauke commented:
As this is in the "Code" section, it makes sense to keep this example as blocks of code (and change the heading accordingly, plus fix the weird start to the description para). Worth introducing a non-code `<pre>` example somewhere else, such as in "Typography" ?

patrickhlauke committed:
Change "preformatted text" heading and intro para

also, stray "blocks" in the "Sample output", which doesn't seem to
belong there.

cvrebert commented:
Yeah, Typography sounds reasonable.
